config/config.go

```diff
@@ -185,6 +185,33 @@ func (config *MixConfig) SetProperty(propertyStr string, value string) error {
 	return errors.Errorf("Property not found in config file: '%s'", property)
 }
 
+// GetProperty parse a property in the format Section.Property, finds the property and returns its
+// current value
+func (config *MixConfig) GetProperty(propertyStr string) (string, error) {
```

Comment:
This duplicates the code in the later PR it seems?

Reply:
It is moved to BaseConfig in that later commit. I din't rewrite the first two commits since I feel they still have a logical progression and that could save me some time. I got nothing against re-working the order though.